semgrep/semgrep/core_output.py

```diff
@@ -354,7 +354,6 @@ def convert_to_rule_match(match: CoreMatch, rule: Rule) -> RuleMatch:
                 start=match.start,
                 end=match.end,
                 extra=match.extra,
-                lines_cache={},
```

Comment:
These were unused so got rid of them as part of refactor to make RuleMatch.lines usable with baseline scanning